packages/react-form-state/src/FormState.ts

```diff
@@ -166,12 +166,18 @@ export default class FormState<
 
     this.setState({submitting: true});
 
-    const result = await onSubmit(formData);
-    if (result) {
-      this.updateRemoteErrors(result);
+    const errors = await onSubmit(formData);
```

Comment:
wait, so `onSubmit` now returns errors or soemthing falsey, whereas before it returned a result?

Reply:
it never returned a result it was just poorly named